lib/experimental/Widgets/Content/TasksList/TaskItem/index.tsx

```diff
@@ -38,18 +42,22 @@ const Wrapper: React.FC<WrapperProps> = ({ onClick, className, children }) => {
   )
 }
 
+export function getWidgetListItemClassname(clickable = false) {
+  return cn(
+    "flex flex-row items-center gap-2 rounded-[8px] border border-solid border-transparent px-2 py-[6px]",
+    clickable
+      ? "cursor-pointer hover:bg-f1-background-tertiary focus:border-f1-background-selected-bold focus:outline-none"
+      : undefined
```

Comment:
I recently changed some styles here for the same component for the widget that we have in the employee profile https://github.com/factorialco/factorial-one/pull/767 I see that you changed the gap, are we sure about this?

Reply:
`py-[6px]` could better be expressed with `py-1.5`